.github/renovate.json

```
@@ -10,6 +10,7 @@
     ":pinOnlyDevDependencies"
   ],
   "labels": ["kind/dependency upgrade"],
+  "baseBranches": ["main", "/^1\\..*\\.x/"],  
```

dependabot will complain about formatting and submit a new PR here, so to save a step:

```suggestion
    "baseBranches": [
    "main",
    "/^1\\..*\\.x/"
  ],
```